2017-March.html

```diff
@@ -0,0 +1,277 @@
+                        <div>
+                            <img class="meetup" src="img/meetup-large.png" alt="Meetup">
+                            <h2 class="fancy">The Venue</h2>
+                        </div>
+
+                        <div>
+                            <img class="white-background" src="img/joindin.png" alt="JoindIn">
+                            <h2 class="fancy">The Speaker</h2>
+                            <a id="joind-in" href="https://joind.in/xxxx"></a>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Conferences and Events</h1>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Yorkshire 2017</h1>
+                    <img style="" width="500" src="img/conference-logos/phpyorkshire-2017.png">
+                    <h4>April 8th</h4>
+                    <h4>York</h4>
+                    <h5>6GBP off: phpams90</h5>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Tour 2017 - Nantes</h1>
+                    <img style="" height="170" src="img/conference-logos/phptour-nantes-2017.png">
+                    <h4>May 18th &amp; 19th</h4>
+                    <h4>Nantes</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHPKonf 2017</h1>
+                    <img style="" height="170" src="img/conference-logos/phpkonf.png">
+                    <h4>May 20th</h4>
+                    <h4>Istanbul</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Srbija Conference 2017</h1>
+                    <img style="" height="170" src="img/conference-logos/phpsrbija-2017.png">
+                    <h4>May 27th &amp; 28th</h4>
+                    <h4>Belgrade</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Dutch PHP 2017</h1>
+                    <img style="" height="170" src="img/conference-logos/dpc-2016.png">
+                    <h4>29th, 30th June &amp; 1st July</h4>
+                    <h4>Amsterdam</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Raffles</h1>
+                </section>
+
+        <section>
+            <p>
+                <img src="img/checkin-to-win.png" alt="Check-in to win"/>
+            </p>
+
+            <a href="http://raffles.amsterdamphp.nl">http://raffles.amsterdamphp.nl</a>
+        </section>
+
+                <section>
+                    <img src="img/logo-transparent-background.png" alt="AmsterdamPHP">
+                    <div class="two-columns" style="margin-top: 1.5em;">
+                        <div>
+                            <h1 id="next-meetup-talk" class="slide fancy" style="font-size: 1.55em;"></h1>
```

Comment:
It's a bit hacky, but with the default `max-width: 200px` the logo is not that visible